scheduler/src/cook/mesos/unscheduled.clj

```diff
@@ -107,7 +108,7 @@
   "IFF the job is not first in the user's queue, returns
   [\"You have x other jobs ahead in the queue\", {:jobs [other job uuids]]}]"
   [conn job]
-  (let [db (d/db conn)
+  (let [db (mt/db conn)
```

Comment:
Nitpick: Should we add a unit tests to verify this behavior change?

Reply:
Done